docs/meetings/20200327-minutes.md

```diff
@@ -0,0 +1,19 @@
+# 27th March 2020 Meeting Minutes
+
+## Attendees
+
+- @subhransu
+- Kajal
```

Comment:
Kajal is @kajalsinha and Sheldon is @SheldonCheng

Reply:
Noted and updated!